gradle/libs.versions.toml

```diff
@@ -54,7 +54,7 @@ coil-gif = { module = "io.coil-kt:coil-gif", version.ref = "coil" }
 coil-compose = { module = "io.coil-kt:coil-compose", version.ref = "coil" }
 
 androidx-appcompat = "androidx.appcompat:appcompat:1.4.1"
-androidx-core = "androidx.core:core-ktx:1.7.0"
+androidx-core = "androidx.core:core-ktx:1.8.0-alpha03"
```

Comment:
What change did you need to pull in the alpha version?

Reply:
Ah, this wasn't strictly necessary, but `ViewCompat.getWindowInsetsController` will be the single non-deprecated way to get the `WindowInsetsControllerCompat` in the upcoming release